application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -33,6 +33,12 @@
 import java.util.function.Supplier;
 import java.util.regex.Pattern;
 
+/**
+ * This command can be used to transfer questions asked in any channel to the helper forum. The user
+ * is given the chance to edit details of the question and upon submitting, the original message
+ * will be deleted and recreated in the helper forum. The original author is notified and redirected
+ * to the new post.
```

Comment:
maybe remove the implementation detail?

Reply:
what do you suggest? could you be specific?

Reply:
```suggestion
 * This command can be used to transfer questions asked in any channel to the helper forum.
```

Reply:
After the rewording its not really implementation detail anymore. It doesnt describe how its done (via modal, via pinging/dm, ...), only a quick overview of the features.